config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:new, :create]
+      post 'like', on: :member
     end
```

Comment:
- [ ] Can you make a route in the config/routes.rb file for creating like? because there should be a route for creating like.

Reply:
- Well resolved @Kccendana 👍🏽